src/ralph/lib/polymorphic/models.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
```

Comment:
add some docstring for this module with in-depth description what is going one here, why we need this mixin etc